src/prebid.js

```diff
@@ -119,11 +120,39 @@ export function setBattrForAdUnit(adUnit, mediaType) {
 function validateBannerMediaType(adUnit) {
   const validatedAdUnit = deepClone(adUnit);
   const banner = validatedAdUnit.mediaTypes.banner;
-  const bannerSizes = validateSizes(banner.sizes);
-  if (bannerSizes.length > 0) {
-    banner.sizes = bannerSizes;
+  const bannerSizes = banner.sizes == null ? null : validateSizes(banner.sizes);
+  const format = adUnit.ortb2Imp?.banner?.format ?? banner?.format;
+  let formatSizes;
+  if (format != null) {
+    deepSetValue(validatedAdUnit, 'ortb2Imp.banner.format', format);
+    banner.format = format;
+    try {
+      formatSizes = format
+        .filter(({w, h, wratio, hratio}) => {
+          if ((w ?? h) != null && (wratio ?? hratio) != null) {
+            logWarn(`Ad unit banner.format specifies both w/h and wratio/hratio`, adUnit);
+            return false;
+          }
+          return (w ?? h ?? wratio ?? hratio) != null;
```

Comment:
shouldn't we check that w AND h aren't null or wratio AND hratio aren't null ? Instead of having just one non null value